disnake/channel.py

```diff
@@ -224,6 +231,19 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
     def members(self) -> List[Member]:
         """List[:class:`Member`]: Returns all members that can see this channel."""
         return [m for m in self.guild.members if self.permissions_for(m).read_messages]
+    
+    @property
```

Comment:
I think we should remove the `@property` decorator here, since this function has a parameter.

Reply:
Yea, fixed it